mappings/net/minecraft/block/entity/EnderChestBlockEntity.mapping

```diff
@@ -1,5 +1,18 @@
 CLASS net/minecraft/class_2611 net/minecraft/block/entity/EnderChestBlockEntity
+	FIELD field_27216 lidAnimator Lnet/minecraft/class_5560;
+	FIELD field_27217 stateManager Lnet/minecraft/class_5561;
+	METHOD <init> (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)V
+		ARG 1 pos
+		ARG 2 state
 	METHOD method_11218 canPlayerUse (Lnet/minecraft/class_1657;)Z
+		ARG 1 entity
```

Comment:
```suggestion
		ARG 1 player
```

Reply:
All of these take an `EntityLike`, not a player.

Reply:
No, the arg type is PlayerEntity (for the 4 comments)